src/main/java/com/josdem/vetlog/model/Pet.java

```diff
@@ -14,84 +14,80 @@
-    private Boolean sterilized = false;
-
-    @Column(nullable = false)
-    private Boolean vaccinated = false;
-
-    @Column(nullable = false)
-    private LocalDateTime dateCreated = LocalDateTime.now();
-
-    @Column(nullable = false)
-    @Enumerated(STRING)
-    private PetStatus status;
-
-    @OneToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "breed_id")
-    private Breed breed;
-
-    @OneToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
-    @JoinColumn(name = "pet_id")
-    private PetAdoption adoption;
-
-    @OneToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "user_id")
-    private User user;
-
-    @OneToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "adopter_id")
-    private User adopter;
-
-    @OneToMany(fetch = FetchType.LAZY)
-    @JoinColumn(name = "pet_image_id")
-    private List<PetImage> images;
-
-    @Transient
-    private List<Vaccination> vaccines;
-}
+ package com.josdem.vetlog.model;
+
+ import static jakarta.persistence.EnumType.STRING;
+ import static jakarta.persistence.GenerationType.IDENTITY;
+ 
+ import com.josdem.vetlog.enums.PetStatus;
+ import jakarta.persistence.CascadeType;
+ import jakarta.persistence.Column;
+ import jakarta.persistence.Entity;
+ import jakarta.persistence.Enumerated;
+ import jakarta.persistence.FetchType;
+ import jakarta.persistence.GeneratedValue;
+ import jakarta.persistence.Id;
+ import jakarta.persistence.JoinColumn;
+ import jakarta.persistence.OneToMany;
+ import jakarta.persistence.OneToOne;
+ import java.time.LocalDateTime;
+ import java.util.List;
+ import lombok.Getter;
+ import lombok.Setter;
+ import lombok.ToString;
+ 
+ @Getter
+ @Setter
+ @Entity
+ @ToString
+ public class Pet {
+ 
+     @Id
+     @GeneratedValue(strategy = IDENTITY)
+     private Long id;
+ 
+     @Column(nullable = false)
+     private String uuid;
+ 
+     @Column(nullable = false)
+     private String name;
+ 
+     @Column(nullable = false)
+     private LocalDateTime birthDate;
+ 
+     @Column(nullable = false)
+     private Boolean dewormed = false;
+ 
+     @Column(nullable = false)
+     private Boolean sterilized = false;
+ 
+     @Column(nullable = false)
+     private Boolean vaccinated = false;
+ 
+     @Column(nullable = false)
+     private LocalDateTime dateCreated = LocalDateTime.now();
+ 
+     @Column(nullable = false)
+     @Enumerated(STRING)
+     private PetStatus status;
+ 
+     @OneToOne(fetch = FetchType.LAZY)
+     @JoinColumn(name = "breed_id")
+     private Breed breed;
+ 
+     @OneToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+     @JoinColumn(name = "pet_id")
+     private PetAdoption adoption;
+ 
+     @OneToOne(fetch = FetchType.LAZY)
+     @JoinColumn(name = "user_id")
+     private User user;
+ 
+     @OneToOne(fetch = FetchType.LAZY)
+     @JoinColumn(name = "adopter_id")
+     private User adopter;
+ 
+     @OneToMany(mappedBy = "pet", fetch = FetchType.LAZY, cascade = CascadeType.ALL) // Map to Vaccination
```

Comment:
I usually avoid to have double relationship, I rely in only one entity definition, please notice we have a relationship defined in `Vaccination` entity: https://github.com/josdem/vetlog-spring-boot/blob/559f732d932ca49c4452153039161d7c9073f45f/src/main/java/com/josdem/vetlog/model/Vaccination.java#L60-L62